### DISCUSSION TITLE
Check if invalid mod happened

### DISCUSSION THREAD

**Discussion by NjuuOwO** [YimMenu/YimMenu#2407] [Q&A]
Is there a way to check if Invalid Mod Crash Detour (or this white box) has happened, and if did log it to console?
![image](https://github.com/YimMenu/YimMenu/assets/45913099/751b0229-ab39-4f76-9b8b-589edb8151ba)

**Comment by lonelybud** [comment]
https://github.com/YimMenu/YimMenu/issues/1909
